examples/Migration/CampaignReportToCsv.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V8.Services;
+using Google.Api.Gax;
+
+namespace Google.Ads.GoogleAds.Examples.V8.Migration
+{
+    /// <summary>
+    /// This code example illustrates how to use Google Ads API to get metrics about a campaign and
+    /// serialize the result as a CSV file.
+    /// </summary>
+    public class CampaignReportToCsv : ExampleBase
+    {
+        private const int PAGE_SIZE = 1000;
```

Comment:
We don't need this pagesize, right?

Reply:
Golden used paged searching. Will remove as I update to streaming search.